client/components/domains/domain-product-price/index.jsx

```diff
@@ -62,9 +63,17 @@ class DomainProductPrice extends React.Component {
 	}
 
 	renderIncludedInPremium() {
-		const textLabel = config.isEnabled( 'domains/kracken-ui' )
-			? this.props.translate( 'Included in Paid Plans' )
-			: this.props.translate( 'Included in WordPress.com Premium' );
+		const { translate, showExpandedPrice } = this.props;
+
+		let textLabel = this.props.translate( 'Included in WordPress.com Premium' );
+
+		const isKrackenUI = config.isEnabled( 'domains/kracken-ui' );
+		if ( isKrackenUI ) {
+			textLabel = showExpandedPrice
+				? translate( 'Price included in paid plans' )
+				: translate( 'Included in paid plans' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Included in Paid Plans' )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).